pkg/providers/common/auditpolicy.go

```diff
@@ -0,0 +1,300 @@
+package common
```

Comment:
Could we pull this out into its own package? Perhaps something already exists that's appropriate?